src/api/custom-schema/type-defs.ts

```diff
@@ -106,6 +106,22 @@ const typeDefs = `
     oldSicCodes: [OldSicCodes]
   }
 
+  type Address {
+    addressLine1: String
+    addressLine2: String
+    town: String
+    county: String
+    postalCode: String
+  }
```

Comment:
Can we update `CompaniesHouseExporterCompanyAddress` to consume this via GQL `implements`?

Reply:
done, will test if it works